keras/layers/rnn/legacy_cells.py

```diff
@@ -186,7 +186,8 @@ def __call__(self, inputs, state, scope=None):
             `2-D Tensor` with shape `[batch_size, self.state_size]`. Otherwise,
             if `self.state_size` is a tuple of integers, this should be a tuple
             with shapes `[batch_size, s] for s in self.state_size`.
-          scope: VariableScope for the created subgraph; defaults to class name.
+          scope: VariableScope for the created subgraph; None uses class name.
```

Comment:
Please quote the None with "`", otherwise the docstring is bit weird to read.

Reply:
Done